tools/lib/provision.py

```diff
@@ -247,12 +247,26 @@ def main(options):
         run(["sudo", "service", "redis-server", "restart"])
         run(["sudo", "service", "memcached", "restart"])
     if not options.is_production_travis:
-        # These won't be used anyway
-        run(["scripts/setup/configure-rabbitmq"])
-        # Need to set up Django before using is_template_database_current.
+
+        # Need to set up Django before using is_template_database_current and rabbitmq credentials
         os.environ.setdefault("DJANGO_SETTINGS_MODULE", "zproject.settings")
         import django
         django.setup()
+
+        import pika
+        from django.conf import settings
+        # These won't be used anyway
```

Comment:
Oh, I see. Yeah, delete this comment. :-)

A comment like this one (in the existing code), that doesn't quite make sense and seems like a non sequitur, is tech debt -- it'll confuse the next reader, just like it confused you. And this kind of comment thrives on being kept around and even moved around as the code changes around it. So in general it's a good practice to take a very skeptical eye to them and try to just delete them.